web/templates/index.html

```diff
@@ -42,13 +42,19 @@ <h6 class="card-subtitle mb-2 text-muted">
                                     <div class="card-text">
                                         <div class="form-group">
                                             <select class="form-control" id="lang1" name="lang1">
+                                                <option selected>{{exampleLanguages.0}}</option>
                                                 {% for lang in languages %}
-                                                <option>{{ lang }}</option>
+                                                    {% if lang != exampleLanguages.0 %}
+                                                        <option>{{ lang }}</option>
+                                                    {% endif %}
                                                 {% endfor %}
                                             </select>
                                             <select class="form-control" id="lang2" name="lang2">
+                                                <option selected>{{exampleLanguages.1}}</option>
                                                 {% for lang in languages %}
-                                                <option>{{ lang }}</option>
+                                                    {% if lang != exampleLanguages.1 %}
+                                                        <option>{{ lang }}</option>
+                                                    {% endif %}
```

Comment:
This can work great, however, it pulls out the random language to the top of the list. Can you move the selected one into the for loop so that they'll still stay alphabetized but that one will still be selected at page start?

Reply:
Can do! :)